src/magic_folder/util/eliotutil.py

```diff
@@ -350,19 +371,56 @@ def get_file():
 _parse_destination_description = _DestinationParser().parse
 
 
-def log_call_deferred(action_type):
+def log_call_deferred(action_type, include_args=False):
     """
     Like ``eliot.log_call`` but for functions which return ``Deferred``.
     """
+
+    if include_args:
+        if include_args is True:
```

Comment:
I guess this less than wonderful bool/list signature is copied from Eliot so it makes sense to keep it instead of doing something less confusing...

Reply:
Not *exactly*; eliot treats `None` as include everything, instead of `True`.